lib/web_ui/test/engine/platform_dispatcher_test.dart

```diff
@@ -12,6 +14,9 @@ import 'package:test/bootstrap/browser.dart';
 import 'package:test/test.dart';
 
 void main() {
+  // Patch browser so that clipboard api is not available.
+  dynamic clipboard = js_util.getProperty(html.window.navigator, 'clipboard');
+  js_util.setProperty(html.window.navigator, 'clipboard', null);
```

Comment:
This is only relevant to a single test. I suggest we do this in the test.

Also, please remember to reset the value of `window.navigator.clipboard`.

Reply:
Done.